server/gitlab/api.go

```diff
@@ -74,6 +74,8 @@ func (g *gitlab) NewGroupHook(ctx context.Context, user *UserInfo, token *oauth2
 		JobEvents:                &webhookOptions.JobEvents,
 		PipelineEvents:           &webhookOptions.PipelineEvents,
 		WikiPageEvents:           &webhookOptions.WikiPageEvents,
+		DeploymentEvents:         &webhookOptions.DeploymentEvents,
+		ReleasesEvents:           &webhookOptions.ReleaseEvents,
```

Comment:
This is called `ReleaseEvents` everywhere else.

Reply:
@wiggin77 `ReleasesEvents` is a key of `AddProjectHookOptions` struct of `github.com/xanzy/go-gitlab`. It is internally defined as `ReleasesEvents`.

Reply:
Can we not at least be consistent within our own code?